src/Host/Client/Impl/Host/BrokerClientProxy.cs

```diff
@@ -26,10 +26,10 @@ internal sealed class BrokerClientProxy : IBrokerClient {
             broker.Dispose();
         }
 
-        public string Name => _broker.Name;
-        public bool IsRemote => _broker.IsRemote;
-        public Uri Uri => _broker.Uri;
-        public AboutHost AboutHost => _broker.AboutHost;
+        public string Name => _broker?.Name;
```

Comment:
Broker should never be null until it is disposed. So do we have calls to broker after its disposal?